.idea/.name

```diff
@@ -0,0 +1 @@
+react-starter-kit-goose
```

Comment:
hard to say. I don't have it locally on my computer. Is it a big deal to leave this file?